python/dgl/distributed/graph_partition_book.py

```diff
@@ -1293,7 +1293,7 @@ def partition_book(self):
 
     def get_data_name(self, name):
         """Get HeteroDataName"""
-        is_node = NODE_PART_POLICY in self.policy_str
+        is_node = self.policy_str.startswith(NODE_PART_POLICY)
```

Comment:
Please also fix `EDGE_PART_POLICY`?

Reply:
In general, we should replace all occurrances of `XXX_PART_POLICY` with `startswith` or create a `is_node_policy` util for that.

Reply:
agree

Reply:
How about enforcing the policy has to be one of `NODE_PART_POLICY` and `EDGE_PART_POLICY`. DGL supports these 2 only for now.

Reply:
Sure. Having more sanity check is always welcomed.

Reply:
added.